homeassistant/components/lektrico/strings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "config": {
+    "flow_title": "{serial_number}",
```

Comment:
By default it will use the name of the integration, which says a lot more than just serial number I guess?

Would recommend to remove this.

```suggestion
```

Reply:
I removed it, thanks.